File: Semester 7/Computational Finance (MA 473)/Term Paper - Python/findAlpha.py
def findAlpha(r, lambda_val, sigma, kappa):
    alpha = -((r - lambda_val*kappa - (1/2)*(sigma*sigma))/(sigma*sigma))
    return alpha


# findAlpha(0.05, 0.1, 0.15, findKappa(-0.9, 0.45))

File: Semester 7/Computational Finance (MA 473)/MA473 Term Paper/Code/findAlpha.m
function alpha = findAlpha(r, lambda, sigma, kappa)
%This function finds value of alpha for CCD scheme
%Input:
%   r       : Interest Rate
%   lambda  : Poisson Arrival Intensity
%   sigma   : std dev of stock prices
%   kappa   : E[q(t)-1]
%Output:
%   alpha
    alpha = -(r-lambda*kappa-sigma*sigma/2)/(sigma*sigma);
    
end